Automattic/jetpack
# Subscriptions: Move the Newsletter module main toggle to separate section in Newsletter settings

Part of https://github.com/Automattic/jetpack/issues/36639

## Proposed changes:

It extracts the Newsletter module main toggle into a separate section.

This is just moving stuff around, no new functionalities have been added.

Diff looks cleaner with the "Hide whitespace" option enabled https://github.com/Automattic/jetpack/pull/37152/files?w=1.

### Before

<img width="1053" alt="Screenshot 2024-05-01 at 11 41 12" src="https://github.com/Automattic/jetpack/assets/4068554/31d2b1a8-971b-4de8-8f4e-b9446e52d9ed">

### After

<img width="1053" alt="Screenshot 2024-05-01 at 11 40 44" src="https://github.com/Automattic/jetpack/assets/4068554/c13535e4-892a-4738-8cb9-0b5ddb3a842a">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Go to Newsletter settings
2. Make sure the main toggle works as expected
